astropy/nddata/nddata.py

```diff
@@ -140,32 +139,32 @@ def __init__(self, data, uncertainty=None, mask=None, wcs=None,
             # other parameters.
             if (unit is not None and data.unit is not None and
                     unit != data.unit):
-                log.info("Overwriting NDData's current "
-                         "unit with specified unit")
+                log.info("overwriting NDData's current "
```

Comment:
To be consistent with the "guidlines" of error messages? (Somewhere someone mentioned that the error messages should start with a lower case as they follow the error type and a colon. However I can't remember who said that and where, but it seems both python and numpy do it that way).

```
In [4]: raise ValueError("This should start lower-case.")
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-4-2a7f08ed1b09> in <module>()
----> 1 raise ValueError("This should start lower-case.")

ValueError: This should start lower-case.
```